pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -178,7 +190,7 @@ func (r *reportingPlugin) Outcome(outctx ocr3types.OutcomeContext, query types.Q
 	// every time since we only want to transmit reports that
 	// are part of the current Query.
 	o.CurrentReports = []*pbtypes.Report{}
-	allExecutionIDs := []string{}
+	var allExecutionIDs []string
```

Comment:
Is this better or cleaner in any way?

Reply:
should be a cleaner way, suggested by the IDE. Though I have no preference.